applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -205,14 +205,14 @@ The ``#XSLMUART`` command manages the UART settings.
 Set command
 -----------
 
-The set command changes the UART settings.
+The set command changes the UART baud rate and hw flow control settings.
 
 Syntax
 ~~~~~~
 
 ::
 
-   #XSLMUART[=<baud_rate>]
+   #XSLMUART=<baud_rate>,<hwfc>
```

Comment:
The AT command needs to be re-designed so it support below
- Only re-config baudrate
- Only re-config HWFC
- Config both baudrate and HWFC
For example, #XSLMUART=<option>,<value>